### PR TITLE
projects/ffmpeg/build.sh: Update for register/codec API change

### LINKED ISSUE #1146: ffmpeg build failing
From https://oss-fuzz-build-logs.storage.googleapis.com/log-cee893d3-914a-4849-b4b3-ed8624db0c55.txt:

```
Step #4: + for c in '$CONDITIONALS'
Step #4: + fuzzer_name=ffmpeg_AV_CODEC_ID_AASC_fuzzer
Step #4: ++ git grep 'REGISTER_[A-Z]*DEC[A-Z ]*(AASC *,' libavcodec/allcodecs.c
Step #4: ++ sed 's/.*, *\([^) ]*\)).*/\1/'
Step #4: + symbol=
Step #4: + echo -en '[libfuzzer]\nmax_len = 1000000\n'
Step #4: + make tools/target_dec__fuzzer
Step #4: make: *** No rule to make target 'tools/target_dec__fuzzer'.  Stop.
Finished Step #4
```

### PR DESCRIPTION
**Pull request by michaelni**
Github: Closes #1146

### CODE REVIEW

File: projects/ffmpeg/build.sh [hunk 1]
```diff
@@ -202,7 +202,8 @@ export TEMP_VAR_CODEC_TYPE="VIDEO"
 CONDITIONALS=`grep 'DECODER 1$' config.h | sed 's/#define CONFIG_\(.*\)_DECODER 1/\1/'`
 for c in $CONDITIONALS ; do
   fuzzer_name=ffmpeg_AV_CODEC_ID_${c}_fuzzer
-  symbol=`git grep 'REGISTER_[A-Z]*DEC[A-Z ]*('"$c"' *,' libavcodec/allcodecs.c | sed 's/.*, *\([^) ]*\)).*/\1/'`
+  symbol=`echo $c | sed "s/.*/\L\0/"`
+#  symbol=`git grep 'REGISTER_[A-Z]*DEC[A-Z ]*('"$c"' *,' libavcodec/allcodecs.c | sed 's/.*, *\([^) ]*\)).*/\1/'`
```
Comment (inferno-chromium) [inline projects/ffmpeg/build.sh]: Can you remove this unneeded line.

**Comment by michaelni** [comment]
@inferno-chromium, The added code, change and commit are under public domain. Is google not allowed to use public domain code ?

**Comment by inferno-chromium** [comment]
@michaelni - i have committed this build fix in https://github.com/google/oss-fuzz/commit/704bca43e494cfe1756dee5a101bf76ff1d4fb6a. Without signing of CLA, we can't accept patches in OSS-Fuzz repository. If you can move the build.sh file to ffmpeg upstream, then you won't run into this issue and you own it, see https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md

**Comment by michaelni** [comment]
@inferno-chromium - thx for commiting. About the CLA, its not that i fundamentally am opposed to signing it (actually i have not really read it at least not recently ...), iam just thinking its absurd to require it and it would be an advantage for google if this requirement was less strict, allowing some kind of standard FOSS license or public domain without having to sign a custom licence

**Pull request closed without merging**